app/admin/members.rb

```diff
@@ -17,6 +17,7 @@
                 :cash_register_proficiency,
                 :register_id,
                 group_ids: [],
+                group_members_attributes: [[%i[id function]]],
```

Comment:
(see DB migration comment)
Please re-word this attribute `assignment` instead of `function`